website/src/pages/index.js

```diff
@@ -0,0 +1,46 @@
+import clsx from 'clsx';
```

Comment:
I noticed several unused imports in your project. Please ensure there are no unused lines. IDEs typically highlight them, and you can automatically remove such imports for the entire folder (just check the available actions).

Reply:
I encountered two imports like this and deleted them :)